src/framework/filterControl/_filterItem.scss

```diff
@@ -45,3 +47,11 @@
         opacity: 1;
       }
     }
+
+.filterFormDropdown--editing {
```

Comment:
That's because `filterFormDropdown` does not know about the editing mode. This style rule adjusts the form to be on top of the `FilterItem`, so I think it belongs here.